rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -851,6 +857,78 @@ static void mouse_pos_to_gem_state(const u32 mouse_no, const gem_config::gem_con
 	pos_to_gem_state(mouse_no, controller, gem_state, mouse.x_pos, mouse.y_pos, mouse.x_max, mouse.y_max);
 }
 
+static bool gun_input_to_pad(const u32 gem_no, be_t<u16>& digital_buttons, be_t<u16>& analog_t)
+{
+  digital_buttons = 0;
+  analog_t = 0;
+
+  if (!is_input_allowed()) {
+    return false;
+  }
+
+  evdev_gun_handler *gh = evdev_gun_handler::getInstance();
+  gh->pool();
+
+  digital_buttons = 0;
```

Comment:
Duplicate code. Can be removed

Reply:
not understood what is duplicated.

Reply:
it's literally done 10 lines above already